KonyAccelerate/forms/mobile/frmAgenda.sm/CopyFlexContainer0a95fcd86c47e42.json

```diff
@@ -0,0 +1,59 @@
+{
```

Comment:
Looks copy skin. Should be removed

Reply:
There are a lot of copy skins. Will take up this task in the last